src/tooltip/test/tooltip.spec.js

```diff
@@ -277,6 +278,52 @@ describe('tooltip', function() {
 
   });
 
+  describe('with specified popup close delay', function () {
+
+    beforeEach(inject(function ($compile) {
+      scope.delay='1000';
+      elm = $compile(angular.element(
+        '<span tooltip="tooltip text" tooltip-popup-delay="1000"' +
+        'tooltip-popup-close-delay="{{delay}}">Selector Text</span>'
+      ))(scope);
+      elmScope = elm.scope();
+      tooltipScope = elmScope.$$childTail;
```

Comment:
Isn't using Angular's internal method `$$childTail` creating a potential for future bugs, should its implementation change in the future?

If I understand it correctly, this is to access child scope, which we should try to avoid to.
Here it is suggested instead to 
> move the model up into the parent...
http://stackoverflow.com/a/14006165/1614973